src/main/scala/beam/utils/map/GpxWriter.scala

```diff
@@ -100,7 +100,7 @@ object GpxWriter {
   }
 
   def drawSourceToDest(outWriter: BufferedWriter, geoUtils: GeoUtils, source: GpxPoint, dest: GpxPoint): Unit = {
-    outWriter.write(s"""<rte>""")
+    outWriter.write("""<rte>""")
```

Comment:
You can also get rid of the triple quotes.

Reply:
done